src/actions/filters.js

```diff
@@ -1,15 +1,9 @@
 import { FILTER_RESET, FILTERS_UPDATE } from './constants'
-import offenses from '../util/offenses'
-import lookupUsa from '../util/usa'
+import filterValidator from '../util/filter'
 
-const isValidCrime = crime => offenses.includes(crime)
 
 export const resetFilter = ({ id }) => ({ type: FILTER_RESET, id })
 export const updateFilters = filters => {
-  const f = { ...filters }
-
-  if (f.crime && !isValidCrime(f.crime)) delete f.crime
-  if (f.place && !lookupUsa(f.place, f.placeType)) delete f.place
-
+  const f = filterValidator({ ...filters })
```

Comment:
Let's combine this with the line below so that we don't have to declare the variable